cmd/massa-wallet/thyra-plugin-wallet.go

```diff
@@ -34,6 +35,10 @@ func startServer(app *fyne.App) {
 	server := restapi.NewServer(massaWalletAPI)
 	server.ConfigureAPI()
 
+	if os.Getenv("STANDALONE") == "1" {
```

Comment:
Non blocking: isn't better have this names mapped in some global constants (if it exists in Go) 🤔 !?

```const STANDALONE = "STANDALONE"```

... and then in the code we could use this global constant instead of the string value it self. This would allow us have only one source of truth for possible future changes instead of replace each occurrence of "STANDALONE".

Feel free to resolve!

Reply:
good catch thanks! I'll do it in the next PR.

Reply:
Two things:
- in Go global constant are only possible for static/compilation time value. here you will have to use a cosntant I guess and having a global constant is a bad practice (in all language). But you could do the same thing using a function returning this value
- I think this is the only place where the value is used, so I guess it's maybe overkill.

Reply:
> ...But you could do the same thing using a function returning this value or a boolean

Ahh, ok! Thank you! Good to know!

> * I think this is the only place where the value is used, so I guess it's maybe overkill.

Indeed!

Reply:
@gregLibert I already implemented here https://github.com/massalabs/thyra-plugin-hello-world/pull/10/files